Testing/SystemTests/tests/analysis/PaalmanPingsMonteCarloAbsorptionTest.py

```diff
@@ -44,6 +44,7 @@ def runTest(self):
             ContainerDensity=6.0,
             CorrectionsWorkspace='flat_plate_corr'
         )
+        SaveNexus(InputWorkspace='flat_plate_corr', Filename='C:\\NewTestData\\CLAYTONirs_PP_MC_flat_plate.nxs')
```

Comment:
Can I ask why we are now saving to NeXus as part of these tests? If this is necessary can we avoid hardcoding OS-dependent paths?

Reply:
I added these lines to save a copy of what would be the new reference files locally, but hadn't realised I'd committed them - I'll get rid